shelley/chain-and-ledger/shelley-spec-ledger-test/test/Test/Shelley/Spec/Ledger/Examples/PoolLifetime.hs

```diff
@@ -629,9 +631,11 @@ aliceRAcnt8 = Coin 11654787878
 bobRAcnt8 :: Coin
 bobRAcnt8 = Coin 1038545454
 
+-- TODO make DeltaCoin?
 deltaT8 :: Coin
 deltaT8 = Coin 317333333333
 
+-- TODO make DeltaCoin?
 deltaR8 :: Coin
 deltaR8 = Coin (-330026666665)
 
```

Comment:
I hope this not being a delta coin is the reason for the test failure! Coins must always be positive or there will be ghosts..

Reply:
changed this to +ve value and subtraction at use site for now!